hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandleWorkflow.java

```diff
@@ -457,6 +458,9 @@ private ValidationResult validate(
 
         // Check if the payer has the required permissions
         if (!authorizer.isAuthorized(payerID, functionality)) {
+            if (functionality == HederaFunctionality.SYSTEM_DELETE) {
```

Comment:
I understand it is "time to commit crimes," but seeing this one hurts. I will create a ticket so we remember to revisit this later.

Reply:
Thank you. Can you post it here? I done it because we want to match the logic as it is in Mono and Richard prefer not to change the test even that system delete and undelete never used in prod enviroments.